react/basic-example/README.md

```diff
@@ -1,11 +1,19 @@
-# talkjs-react-example
+# Getting Started with the react example app
 
-A simple React project using `talkjs`, bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+This project is designed to demonstrate how you can easily add chat in to your React application using [TalkJS](https://talkjs.com)
 
-Open `/src/index.js ` and replace YOUR_APP_ID with your own appId which you can find in the [TalkJS dashboard](https://talkjs.com/dashboard).
+![Chatbox Example](public/example.png)
 
-Run the following commands to start the example:
-```
-npm install && npm run start
+First you'll need to enter your own `appId` inside the `App.js` component. You can find your `appId` in the [TalkJS dashboard](https://talkjs.com/dashboard)
+
+To run the example, run `yarn start` and open [http://localhost:3000](http://localhost:3000) in the browser.
+
+The page will reload if you make edits.
```

Comment:
It might be worth mentioning somewhere that this example was created with Create React App, but that it ought to be useful for any type of React project